sources/packages/web/src/components/aest/institution/modals/ApproveProgramModal.vue

```diff
@@ -12,11 +12,18 @@
       ></formio>
     </template>
     <template v-slot:footer>
-      <footer-buttons
-        primaryLabel="Approve now"
-        @primaryClick="approveProgram"
-        @secondaryClick="dialogClosed"
-      />
+      <check-a-e-s-t-permission-role
+        :role="Role.InstitutionApproveDeclineProgram"
+      >
+        <template v-slot="{ isReadonly }">
```

Comment:
I think we once discussed about using the short cut instead of v-slot for template.